src/sonic-build-hooks/scripts/utils.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+# Lock macro for shared file access
+# Parameters:
+#  $(1) - Lock file name
+#  $(2) - Timeout value
+function FLOCK()
+{
+	local filename=$(dirname $1)
+    local timeout=${2:-360}
```

Comment:
Could you please fix the tab/space format issue?

Reply:
done